keras_nlp/src/models/video_swin/video_swin_backbone_test.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2024 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import keras
+import numpy as np
+import pytest
+from keras import ops
+
+from keras_nlp.src.models.video_swin.video_swin_backbone import (
+    VideoSwinBackbone,
+)
+from keras_nlp.src.tests.test_case import TestCase
+
+
+class VideoSwinSBackboneTest(TestCase):
```

Comment:
Rewrite this test case to follow the standard for other backbones. Use the common routines for testing backbones and saving etc. `run_backbone_test`